examples/audio/md/eeg_signal.md

```diff
@@ -0,0 +1,1008 @@
+# Electroencephalogram Signal Classification for action identification
+
+**Author:** [Suvaditya Mukherjee](https://github.com/suvadityamuk)<br>
+**Date created:** 2022/11/03<br>
+**Last modified:** 2022/11/05<br>
+**Description:** Training a Convolutional model to classify EEG signals produced by exposure to certain stimuli.
+
+
+<img class="k-inline-icon" src="https://colab.research.google.com/img/colab_favicon.ico"/> [**View in Colab**](https://colab.research.google.com/github/keras-team/keras-io/blob/master/examples/audio/ipynb/eeg_signal.ipynb)  <span class="k-dot">•</span><img class="k-inline-icon" src="https://github.com/favicon.ico"/> [**GitHub source**](https://github.com/keras-team/keras-io/blob/master/examples/audio/eeg_signal.py)
+
+
+
+---
+## Introduction
+
+The following example explores how we can make a Convolution-based Neural Network to
+perform classification on Electroencephalogram signals captured when subjects were
+exposed to different stimuli.
+We train a model from scratch since such signal-classification models are fairly scarce
+in pre-trained format.
+The data we use is sourced from the UC Berkeley-Biosense Lab where the data was collected
+from 15 subjects at the same time.
+Our process is as follows:
+
+- Load the [UC Berkeley-Biosense Synchronized Brainwave Dataset](https://www.kaggle.com/datasets/berkeley-biosense/synchronized-brainwave-dataset)
+- Visualize random samples from the data
+- Pre-process, collate and scale the data to finally make a `tf.data.Dataset`
+- Prepare class weights in order to tackle major imbalances
+- Create a Conv1D and Dense-based model to perform classification
+- Define callbacks and hyperparameters
+- Train the model
+- Plot metrics from History and perform evaluation
+
+This example needs the following external dependencies (Gdown, Scikit-learn, Pandas,
+Numpy, Matplotlib). You can install it via the following commands.
+
+Gdown is an external package used to download large files from Google Drive. To know
+more, you can refer to its [PyPi page here](https://pypi.org/project/gdown)
+
+---
+## Setup and Data Downloads
+
+First, lets install our dependencies:
+
+
+```python
+!pip install gdown
+!pip install sklearn
+!pip install pandas
+!pip install numpy
+!pip install matplotlib
+```
+
+<div class="k-default-codeblock">
+```
+Requirement already satisfied: gdown in /home/suvaditya/Programming/personal/oss/oss-ml/lib/python3.8/site-packages (4.5.3)
```

Comment:
lets remove this codeblock and add `-q` to the pip commands.

Reply:
Alright, shall do that.